recipes-dom0/dom0-image-rescue-initramfs/dom0-image-rescue-initramfs.bbappend

```diff
@@ -15,11 +15,14 @@ SRC_URI = " \
 # N.B. xt_shared_env.inc MUST be listed AFTER meta-xt-prod-extra
```

Comment:
This comment line need to be rewritten, taking into account removal of `xt_shared_env.inc`.

Reply:
agree